index.html

```diff
@@ -1961,6 +1964,13 @@ <h4 id="maturity-levels">6.1.2 Maturity Levels</h4>
           abandoned without producing a Recommendation.</dd>
       </dl>
 
+      <p>The decision to advance to a new maturity level must be made solely based on the technical maturity of the work.
+        <strong>Note:</strong> Should faster advancement to meet scheduling considerations be desired,
+        this may be achieved by reducing the scope of the technical report to a subset that is adequately mature and deferring
+        less stable features to other technical reports.</p>
+
+
+
```

Comment:
Disagree with this addition, as it reflects hope over realpolitik. A group may rationally and sensibly opt to bundle features and hold back "technically mature" features in order to offer a synchronized package of mature features.

Reply:
NB: that disagreement refers to the whole insertion, " <p>The decision to advance to a new maturity level must be made solely based on the technical maturity of the work. 
        <strong>Note:</strong> Should faster advancement to meet scheduling considerations be desired, this may be achieved by reducing the scope of the technical report to a subset that is adequately mature and deferring less stable features to other technical reports.</p>"

Reply:
I don't understand what you're saying. (a) you think groups should push immature features to CR? or (b) they should hold back and wait until all they stuff they wants is mature (which doesn't conflict with this statement) or (c) something else?

Reply:
I am also confused: I don't understand how your point @wseltzer differs from the proposed text. It seems that we are all in agreement that, in the case that quicker advancement is desired, since advancement is based solely on the technical maturity, then insufficiently mature features would need to be dropped to achieve that desire. If alternatively the desire is to ship with those features, then the converse is that the group must wait until they are sufficiently mature. Scope and timing are both within the control of the working group, but may work against each other. Does the wording need changing to make it even clearer that this is what the note says?

Reply:
Correct, @nigelmegitt , and in the case the group decides that bundling is more important than speed, they're making decision (not) to advance based on factors other than solely technical maturity.

Reply:
We don't say anything about why a group might NOT advance; it might be based on chair incompetence, sheer stupidity, utter laziness, or a host of factors. We only set requirements on the decision TO advance.

Reply:
Nonetheless, I find the addition unduly constraining.

Reply:
I just don't get it:
* It is a note, so it is not constraining
* it says that the group has a choice between either waiting for everything to be ready and advancing together, or advancing some parts first and the rest later if fast advancement is desired and not everything is ready. Disagreeing with that seems to be calling for advancing things that are not ready, which I strongly object to.

(I don't object to removing the note for the sake of making progress, since it is indeed just a note, unless the intent is to allow advancing things that are not ready).

Reply:
Maybe I am just dense, but what alternative is there other than:
1. advancing parts that are ready first, other parts later
2. waiting until everything is ready, then advancing everything
3. advancing things that are not ready.

The note says: ”if you don't want to do 2, you can do 1“. Unless you're claiming that 3 is a legitimate thing to do, I just don't understand the pushback.

Reply:
Suggest s/may/can/ in the note, but otherwise agreed with frivoal/nigel/dsinger : unless you're arguing for #3 (advancing things that are not ready in pursuit of preconceived deadlines) I don't understand the objection.

Reply:
> Suggest s/may/can/ in the note

Good suggestion, which I don't expect to change the opinion of anyone already in favor of this, so I've updated the text accordingly: https://github.com/w3c/w3process/pull/214/commits/f5e96b21e43d97cb176974c88dc3cfdb14dcc3f5